hotkey.js

```diff
@@ -14,12 +16,41 @@ function ignoreHotkeysIn(target) {
   )
 }
 
+function choseTheElement(list, current) {
+  for (let j = 0; j < list.length; j++) {
+    let closestElement = list[j].closest(`[${IGNORE_ATTR}]`)
+    if (closestElement === current || closestElement == null) {
```

Comment:
```suggestion
    if (closestElement === current || !closestElement) {
```

It should be smaller